tracing-attributes/src/lib.rs

```diff
@@ -211,20 +211,21 @@ pub fn instrument(
     let input: ItemFn = syn::parse_macro_input!(item as ItemFn);
     let args = syn::parse_macro_input!(args as InstrumentArgs);
 
+    let span_declared_name = input.sig.ident.to_string();
```

Comment:
This naming is not super clear to me...also, this is the name of the instrumented function, and not _necessarily_ the name of the span itself: it may be overridden with `name = "something_else"` in the macro's arguments.